setup.py

```diff
@@ -30,25 +31,28 @@ def local_open(fname):
 for directory in os.walk('example/templates'):
     example_dirs.append(directory[0][8:] + '/*.*')
 
+with local_open(os.path.join('pages', '__init__.py')) as f:
+    exec(f.read(), {}, pages)
+
 url_schema = 'http://pypi.python.org/packages/source/d/%s/%s-%s.tar.gz'
-download_url = url_schema % (package_name, package_name, pages.__version__)
+download_url = url_schema % (package_name, package_name, pages['__version__'])
 
 install_requires = [
-    'django-mptt>=0.8.3,<0.9.0',
-    'six>=1.10.0,<1.11.0',
-    'Pillow>=3.2.0,<3.3.0',
-    'tqdm>=4.4.0,<4.5.0',
-    'django-taggit>=0.18.1,<0.19.0',
-    'requests>=2.9.0,<3.0.0',
+    'django-mptt>=0.8.3',
+    'six>=1.10.0',
+    'Pillow>=3.2.0',
+    'tqdm>=4.4.0',
+    'django-taggit>=0.18.1',
+    'requests>=2.9.0',
```

Comment:
No probs I am all relaxed ;-) just not quite convinced. The problem is new packages arise that break compatibility happen from time to time (it happened in the past). If that happens I have no way to know it and the stuff I just put in the documentation will stop to work:

`$ pip install django-page-cms[full]`

How do I deal with that? Do I need to go back to the requirements file for install instruction? It seems like step backward.